test/simple_arena_allocator.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <memory>
```

Comment:
it will not fix gcc 4.9 build because there is no std::aligned provided by this compiler :( You need to use this header https://github.com/VestniK/portable_concurrency/blob/master/portable_concurrency/bits/align.h which provides workaround.